### DISCUSSION TITLE
Mendapatkan nilai longitude dan latitude pada leaflet map di shiny

### DISCUSSION THREAD

**Discussion by MF-Faqih** [teamalgoritma/community#680] [Q&A] [answered]
Saya membuat sebuah map sederhana kota jakarta yang mana menggunakan batas (border) kota administratif (Jakarta selatan, Jakarta Barat dan seterusnya). Adakah cara tertentu agar saya bisa mendapatkan nilai longitude dan latitude ketika melakukan klik pada salah satu area dan menyimpannya pada variable terpisah? Variabel ini nantinya akan digunakan untuk tujuan lainnya seperti melakukan filter pada data frame yang ditampilkan pada halaman yang sama dengan map. Terimakasih banyak.

**Comment by jafarhf1** [comment, marked as answer]
Dear @MF-Faqih,

Terkait dengan pertanyaannya, Apakah output yang diinginkan mirip dengan yang ditampilkan pada dashboard ini ? https://ibm-dse-rshiny.shinyapps.io/Example-2-Leaflet-Capture-Mouse-Events/

Apabila seperti itu, maka Bapak bisa mencoba melihat step by step pembuatannya pada artikel berikut ini : https://medium.com/ibm-data-ai/capture-and-leverage-mouse-locations-and-clicks-on-leaflet-map-6d8601e466a5

Adapun untuk bisa mendapatkan longitude dan latitude dari area jakarta, maka bisa mencoba untuk mencari file geojson mengenai hal tersebut. File ini biasanya sudah dikumpulkan di beberapa github salah satunya adalah berikut ini: https://github.com/thetrisatria/geojson-indonesia/blob/master/province/id-jk.geojson?short_path=273c5e7

Demikian yang dapat kami sampaikan, Semoga membantu.

**Reply by MF-Faqih** [reply]
Terimakasih, benar seperti itu yang saya butuhkan
